mmpose/core/evaluation/top_down_eval.py

```diff
@@ -272,3 +272,12 @@ def keypoints_from_heatmaps(heatmaps,
         preds[i] = transform_preds(preds[i], center[i], scale[i], [W, H])
 
     return preds, maxvals
+
+
+def crowd_keypoints_from_heatmaps(heatmaps,
+                                  center,
+                                  scale,
+                                  post_process=True,
+                                  unbiased=False,
+                                  kernel=11):
+    pass
```

Comment:
evaluation will be added in separate pr, right?

Reply:
will be added in this pr